### PR TITLE
Lj 390 card fixes

### PR DESCRIPTION
**Pull request by MarcoGoC**
# Description

[LJ-390] Card Fixes

Removed the alt test as the images are for decoration only
Added underline to the links  

## Test Instructions

1. Download, run and verify changes 

## Meets Definition of Done

- [x] Meets design spec
- [ ] unit tests are included

[LJ-390]: https://lj-decd.atlassian.net/browse/LJ-390

### CODE REVIEW

**Review by krischarbonneau — APPROVED** [review]
Looks good to me!

File: components/atoms/Card.js [hunk 1]
```diff
@@ -9,13 +8,7 @@ export default function Card(props) {
...
     <div className="text-black text-left card rounded-tl-md rounded-b-md border my-6 mx-4">
       <div>
-        <Image
-          src={props.image}
-          alt={props.imgalt}
-          width={252}
-          height={101}
-          layout="intrinsic"
-        ></Image>
+        <img src={props.image} width={252} height={101} layout="intrinsic" />
```
Comment (krischarbonneau) [inline components/atoms/Card.js:11]:
```suggestion
        <img src={props.image} width={252} height={101} layout="intrinsic" alt={props.imgalt} />
```
Maybe adding this back will fix the issue? Then a null image alt will get passed in and axe will recognize it as a decorative image

Comment (krischarbonneau) [inline components/atoms/Card.js:11]:
```suggestion
        <img src={props.image} width={252} height={101} layout="intrinsic" alt={props.imgalt} />
```
Maybe adding this back will fix the issue? Then a null image alt will get passed in and axe will recognize it as a decorative image

Comment (MarcoGoC) [reply components/atoms/Card.js]: added alt=""

Comment (krischarbonneau) [reply components/atoms/Card.js]: Looks good to me now. LGTM

**Pull request merged**